readthedocs/projects/migrations/0050_migrate_external_builds.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.2.10 on 2020-05-19 13:24
+
+from django.db import migrations
+
+
+def migrate_features(apps, schema_editor):
+    # Enable the PR builder for projects with the feature flag
+    Feature = apps.get_model('projects', 'Feature')
+    for project in Feature.objects.get(feature_id='external_version_build').projects.all():
```

Comment:
I think you can just use an `.update` instead of iterate over each project.

Reply:
It's a migration, it's ok for it to be slow :)